docs/modules/sql/pages/select-statement.adoc

```diff
@@ -97,12 +97,12 @@ FROM
 ORDER BY salary DESC
 LIMIT 5;
 ----
-
-== Unsupported Features
-
-The following features are *not supported* and are planned for future releases:
-
-* `GROUP BY`/`HAVING`
-* `JOIN`
-* set operators (`UNION`, `INTERSECT`, `MINUS`)
+
+== Unsupported Features
+
+The following features are *not supported* and are planned for future releases:
+
+* `GROUP BY`/`HAVING`
+* `JOIN`
+* set operators (`UNION`, `INTERSECT`, `MINUS`)
```

Comment:
Not sure why I have a diff here. Did not change this part

Reply:
This might be due to CRLF and LF difference (line endings difference between linux and windows). Which one do you use?

Reply:
This is due to the fact that select-statement.adoc has CRLF at the end of the file but has LF in the other lines